packages/vue3-vite/src/util/defaults.ts

```diff
@@ -38,53 +38,56 @@ export const CypressDevDependencies = {
 };
 
 export const VitestDevDependencies = {
-  '@nrwl/vite': '^15.4.2',
+  '@nrwl/vite': '^15.4.4',
   '@vue/test-utils': '^2.2.6',
   'happy-dom': '^8.1.1',
-  vitest: '^0.25.8',
+  vitest: '^0.26.3',
 };
 
 export const JestDevDependencies = {
+  '@babel/preset-env': '^7.20.2',
+  '@jest/globals': jestGlobalsVersion,
   '@vue/vue3-jest': vue3JestVersion,
   'jest-environment-jsdom': '^29.3.1',
 };
 
+export const LintDevDependencies = {
+  '@intlify/eslint-plugin-vue-i18n': '^2.0.0',
+  '@vue/eslint-config-typescript': '^11.0.2',
+  eslint: eslintVersion,
+  'eslint-config-prettier': '^8.6.0',
+  'eslint-plugin-import': '^2.26.0',
+  'eslint-plugin-vue': '^9.8.0',
+  stylelint: stylelintVersion,
+  'stylelint-config-standard': stylelintConfigVersion,
+  '@typescript-eslint/eslint-plugin': '5.48.0',
+  '@typescript-eslint/parser': '5.48.0',
+  'vue-eslint-parser': '^9.1.0',
+};
+
 export const ProjectDependencies = {
   vue: vueVersion,
   'vue-i18n': vueI18nVersion,
   'vue-router': vueRouterVersion,
   'date-fns': '^2.29.3',
-  '@sampullman/fetch-api': '^0.8.1',
+  '@sampullman/fetch-api': '^0.9.1',
 };
 
 export const ProjectDevDependencies = {
   vite: viteVersion,
-  eslint: eslintVersion,
   picocolors: '1.0.0',
   'class-transformer': '^0.5.1',
-  '@intlify/eslint-plugin-vue-i18n': '^2.0.0',
-  'eslint-plugin-import': '^2.26.0',
-  'eslint-plugin-vue': '^9.8.0',
-  'vue-eslint-parser': '^9.1.0',
-  '@vue/eslint-config-typescript': '^11.0.2',
-  '@vue/compiler-sfc': vueVersion,
```

Comment:
It seems like we don't need to include `@vue/compiler-sfc` any more, it comes with `vue`. I remember reading some Vue release notes about this, but can't find it now.